src/templates/validate.yaml

```diff
@@ -2,20 +2,19 @@ apiVersion: admissionregistration.k8s.io/v1beta1
 kind: ValidatingWebhookConfiguration
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: seldon-system/seldon-serving-cert
+    cert-manager.io/inject-ca-from: kubeflow/seldon-serving-cert
   creationTimestamp: null
   labels:
     app: seldon
-    app.kubernetes.io/instance: seldon1
-    app.kubernetes.io/name: seldon
-    app.kubernetes.io/version: v0.5
-  name: seldon-validating-webhook-configuration
+    app.kubernetes.io/instance: 'seldon-core'
+    app.kubernetes.io/name: 'seldon-core-operator'
+    app.kubernetes.io/version: '1.6.0'
+  name: seldon-validating-webhook-configuration-kubeflow
 webhooks:
 - clientConfig:
-    caBundle: Cg==
+    caBundle: 'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lSQU5sU0g0RzU4UEpMRnd4TmVpSGM2bE13RFFZSktvWklodmNOQVFFTEJRQXcKSERFYU1CZ0dBMVVFQXhNUlkzVnpkRzl0TFcxbGRISnBZM010WTJFd0hoY05NakV3TmpJeE1UTXlPVE15V2hjTgpNakl3TmpJeE1UTXlPVE15V2pBY01Sb3dHQVlEVlFRREV4RmpkWE4wYjIwdGJXVjBjbWxqY3kxallUQ0NBU0l3CkRRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFKOXJnOGVrN2lXc3cxYWt1WHh0TWJxU1FKbnYKUFR4S1FKTGFseGZmWUZ0ZGxtMnN6ZWhMK3I2V0w0ZDdZM2R2d2lwZFc4U2o1TkMrQ0NBT0grK0RPT2FNVkF0SApuYzA1V3VkV1ZucGxIeDRtdnFWeEgrWWpudzRhSjFISS9XVW16dDVmYkp3aWx6dFJGYWV5NU81ZGJibTd5SkxmCmdhMDdkRFJHOExQbVR3NkFyR1oyMkt4djF3c202Rks1WlczcHpTL0VvSHZoNUtvME5lNTI0V28rem9JWWtuNlgKWmNuSlBaNFk5d01rR3FVKzBqRnk5TU9maFNJSTE2TlB0U2QraDAyS292VVQ5U3pxSVNrNUFmUkN5V0ZSeE9NYQpoUEtqZWRmV1gzOENzT2NETGdxWk1wbHJxdy9pQmZYY2tZVmpJakpEUk1mL3oxQWhpTnJMQm1UbXRIMENBd0VBCkFhTkNNRUF3RGdZRFZSMFBBUUgvQkFRREFnS2tNQjBHQTFVZEpRUVdNQlFHQ0NzR0FRVUZCd01CQmdnckJnRUYKQlFjREFqQVBCZ05WSFJNQkFmOEVCVEFEQVFIL01BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQlZveFNzcWtoVwpyRnNhRldSSTFsVEpKSlRLRkFHalkwSXM1QlBieHpNbGVtK3dpRzdKWVg1NUJDeUlMWkVoNlJxbWJaQlpuRng4CmlDSzNVT3lUdmZoQmN0RUpiUGd4WldtTnFCd04ra2lYWE5maWNjdmQwSEs3eTBJRnEvU2FqSklyMkJMbHBiMHEKWk5WRzYvSUlmNXJJWlpZTmdsVThPbmtRblZmVVJsandPL3NlTHVOTzAyNWZ5MmNzV2R1MFRwb2pNRDJhdjBEdwp2Wm9LMUtjSFFLZ2t5eDhZakRXQ1dpbFl2OXpHWDRzejRsaDV2QW01d2owRW5nVGhCclg4a2gvK3JNNFpiQmIwCmlsaSt1OTNpaXVTcHFjVU1UV3lWUU1lU1NBU2dobnZyMytJL3lnNE00OGRRUk1ubkI2dkJHejJ6K2JaUGhFa0QKMTdMaDg3K3ZCSERzCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K'
```

Comment:
We should probably revert all of these `caBundle` changes back to `Cg==`. I believe this `caBundle` value will get overwritten, but the `Cg==` value makes it much more obvious when something goes wrong. We'll probably just have to remember to manually do this every time we update these files.

Reply:
+1, I wasn't sure if that was intentional in the last version of this file. Will change it.

Reply:
This is done.